### ISSUE TITLE
Error with 'useReduxContext' in Next.js 13.4.11: Cannot destructure property 'store' as it is null.

### ISSUE THREAD

**Issue by VladSturm** [vercel/next.js#53001]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.5.0: Mon Apr 24 20:53:44 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T8103
    Binaries:
      Node: 20.2.0
      npm: 9.6.6
      Yarn: 1.22.19
      pnpm: 8.6.0
    Relevant Packages:
      next: 13.4.11
      eslint-config-next: 13.4.11
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.6
    Next.js Config:
      output: standalone
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Standalone mode (output: "standalone")

### Link to the code that reproduces this issue or a replay of the bug

Private Project

### To Reproduce

next build

### Describe the Bug

I am facing an issue with the 'useReduxContext' hook in Next.js version 13.4.11. The problem seems to be related to accessing the 'store' property, which is returning null unexpectedly. No build issues found in version 13.4.9

### Expected Behavior

Successful "next build"

**Comment by thucne** [comment]
I faced the same issue, I had to downgrade next

**Comment by VladSturm** [comment]
the problem was solved in version 13.4.12

**Comment by AniruddhaGawali** [comment]
No the bug is still there in 13.4.12 also
